app/Services/Model/Imp/SummitTicketTypeService.php

```diff
@@ -350,6 +369,32 @@ public function getAllowedTicketTypes(Summit $summit, Member $member): array
                 )
             );
 
+            $promocode = null;
+
+            if (!is_null($filter)) {
+                $element = $filter->getUniqueFilter('promo_code');
+                $promocode_id = intval($element->getRawValue());
```

Comment:
the value is not an id
is the promo code itself promo_code == promocode.code

Reply:
done